spec/solver_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../solver'
+
+describe Solver do
+  subject { Solver.new }
+
+  it 'should return the factorial of 6' do
+    expect(subject.factorial(6)).to eq(720)
+  end
+
+  it 'should return the factorial of 5' do
+    expect(subject.factorial(5)).to eq(120)
+  end
```

Comment:
Great job so far. But based on the project requirement it would be better if you write the test for zero and negative numbers. Based on the case when the input is 0 the result should be 1 and when the input is a negative number the result should raise an exception. Kindly fix it

![image](https://github.com/Javivy/tdd-ruby/assets/101523582/40875506-cbc3-4523-bf8d-3a704e2ff19f)